test/powershell/engine/Basic/DefaultConfirmImpact.Tests.ps1

```diff
@@ -0,0 +1,256 @@
+            @{ Cmdlet = 'Set-Service'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Set-StrictMode'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Set-TimeZone'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Set-TraceSource'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Set-Variable'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Show-Markdown'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Sort-Object'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Split-Path'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Start-Job'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Start-Process'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Start-Service'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Start-Sleep'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Stop-Computer'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Stop-Job'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Stop-Process'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Stop-Service'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Suspend-Service'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Tee-Object'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-Connection'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-Json'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-ModuleManifest'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-Path'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Test-PSSessionConfigurationFile'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Trace-Command'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Unblock-File'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Unregister-Event'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Unregister-PSSessionConfiguration'; ConfirmImpact = 'Low' }
+            @{ Cmdlet = 'Update-FormatData'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Update-Help'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Update-TypeData'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Wait-Debugger'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Wait-Event'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Wait-Job'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Wait-Process'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Where-Object'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Debug'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Error'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Host'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Information'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Output'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Progress'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Verbose'; ConfirmImpact = 'Medium' }
+            @{ Cmdlet = 'Write-Warning'; ConfirmImpact = 'Medium' }
```

Comment:
I'd reformat the array to align like:
```suggestion
            @{ Cmdlet = 'Write-Progress';          ConfirmImpact = 'Medium' } 
            @{ Cmdlet = 'Write-Verbose';           ConfirmImpact = 'Medium' }
            @{ Cmdlet = 'Write-Warning';          ConfirmImpact = 'Medium' }
```

Reply:
I had it like this at one point, but I guess I must have had to copy paste it at some point an VS Code decided it wasn't needed. Fixed! 😄